packages/documentation-site/patternfly-docs/content/design-guidelines/styles/motion/Animations/Animations.md

```diff
@@ -0,0 +1,55 @@
+---
+id: Motion
+section: design-foundations
+source: demo
+---
+
+import { Fragment, useRef, useState, useEffect, useCallback } from 'react';
+
+import BellIcon from '@patternfly/react-icons/dist/esm/icons/bell-icon';
+import CogIcon from '@patternfly/react-icons/dist/esm/icons/cog-icon';
+import BarsIcon from '@patternfly/react-icons/dist/js/icons/bars-icon';
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+import PowerOffIcon from '@patternfly/react-icons/dist/esm/icons/power-off-icon';
+import QuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/question-circle-icon';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+import imgAvatar from '@patternfly/react-core/src/components/assets/avatarImg.svg';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import pfLogo from '@patternfly/react-core/src/demos/assets/PF-HorizontalLogo-Color.svg';
+import MultiContentCard from "@patternfly/react-component-groups/dist/dynamic/MultiContentCard";
+import { ArrowRightIcon, LockIcon, PortIcon, CubeIcon, AutomationIcon, ExclamationCircleIcon, CheckCircleIcon, ExclamationTriangleIcon, HamburgerIcon, TimesIcon} from '@patternfly/react-icons';
+import { createUseStyles } from 'react-jss';
```

Comment:
is use of react-jss necessary?
Is it used anywhere in this demo

Reply:
It is used by the `MultiContentCard`. I'm not sure if we need it imported here. I can try removing it.